### DISCUSSION TITLE
Question about `artistId` for multi-artist album/songs

### DISCUSSION THREAD

**Discussion by deluan** [opensubsonic/open-subsonic-api#93] [General]
Now that OS introduced multiple artists with roles, what is expected to be returned in the `AlbumID3` response's `artistId` field, if an album contains multiple album artists? Same applies for the `Child` response, when a song has multiple track artists? 

How other servers are handling this? Omitting the artistId in this case? Or leaving it in place for legacy clients? In that case, whose ID is it for? The firs artist in the list?

**Comment by lachlan-00** [comment]
it's useful for people who haven't implemented multi artist.

i use it as the primary artist so it's the main artist and the rest are related in the list.

**Reply by deluan** [reply]
That's what I'm planning to do, but there are various situations where there is no clear "primary" artist. Ex: https://musicbrainz.org/release/a62fe4ec-34e7-450a-8ab0-a1a55c6d2287?tport=8000

I think I'll just get the first from the list and ask users to migrate to a newer client :)

**Reply by lachlan-00** [reply]
that's it for me too, first mbid / artist in the list is primary.

**Reply by Tolriq** [reply]
The initial call was that the value was not supposed to change, most servers already supported multiple artists internally. 
Since you are migrating to something new, you can as you want as long as you return something a little coherent for legacy clients.
